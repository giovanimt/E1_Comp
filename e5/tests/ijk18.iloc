addI rsp, 4 => rsp
loadI 10 => r0
addI rfp, 0 => r1
store r0 => r1
L2: nop 
addI rfp, 0 => r2
load r2 => r3
loadI 5 => r4
cmp_GT r3, r4 => r5
cbr r5 => L0, L1
L0: nop 
addI rfp, 0 => r6
load r6 => r7
loadI 1 => r8
sub r7, r8 => r9
addI rfp, 0 => r10
store r9 => r10
jumpI  => L2
L1: nop 
