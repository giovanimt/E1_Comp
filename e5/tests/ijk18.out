00038762 5
